docs/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
This page doesn't have much content. Is this intended for GitHub pages?

Reply:
It's 530K of javascript containing the 3D example, which seemed the most interesting/relevant for an elaborate example.